docs/UserGuide.md

```diff
@@ -3,271 +3,443 @@ layout: page
+* Note that your URLs with should not contain `>` characters either! They are not valid URLs by the Internet's definition.
 
-Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+**Examples**:
+* `list` followed by `policy 2` displays the policies associated with the 2nd person in the address book.
+  
+  ![with policy URL](images/with_policy_URL.png)
+  
+* `find n/Bernice` followed by `policy 1` displays the policies associated with the 1st person in the results of the `find` command.
+  
+  ![with policy URL](images/find-then-policy.png)
 
-### Viewing a contact's policies: `policy`
 
-Launches a popup window to show all the policies associated with the selected contact, if the selected contact has any policies.
 
-![without policy URL](images/without_policy_URL.png)
-![with policy URL](images/with_policy_URL.png)
+### Delete client: `delete`
 
-Format: `policy INDEX`
+**Purpose**: Deletes the specified client from ClientBook after listing or finding contacts.
 
-* Selects the client at the specified `INDEX`.
+**Format**: `delete INDEX`
+
+* Deletes the client at the specified `INDEX`.
 * The index refers to the index number shown in the displayed client list.
 * The index **must be a positive integer** 1, 2, 3, …​
-* Policy ID cannot contain '>' character!
-* Note that your URLs with should not contain '>' characters either! They are not valid URLs by the Internet's definition.
 
-Examples:
-* `list` followed by `policy 2` displays the policies associated with the 2nd person in the address book.
-* `find Betsy` followed by `policy 1` displays the policies associated with the 1st person in the results of the `find` command.
+**Examples**:
+* `list` followed by `delete 2` deletes the 2nd person in the address book.
 
-### Sorting the list of clients : `sort`
+  ![delete 2](images/delete-2.png)
 
-Sorts the list of clients in ClientBook.
+* `find n/Charlotte` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
-Format: `sort ATTRIBUTE DIRECTION`
+  ![delete charlotte](images/delete-charlotte.png)
 
-* Sorts the list of clients according to the specified `ATTRIBUTE` and `DIRECTION`.
-* The specified `ATTRIBUTE` can be `-n` to sort by name or `-i` to sort by insurance policy.
-* The specified `DIRECTION` can be `-asc` for ascending order or `-des` for descending order.
 
-Examples:
-* `sort -n -asc` sorts the list of clients by name in ascending order.
+### Sort list of clients: `sort`
 
-### Exiting the program : `exit`
+**Purpose**: Sorts the current list of clients in ClientBook.
+
+**Format**: `sort -ATTRIBUTE -DIRECTION`
 
-Exits the program.
+* Sorts the list of clients according to the specified `ATTRIBUTE` and `DIRECTION`.
+* The specified `ATTRIBUTE` can be `-n` to sort by name or `-i` to sort by insurance policy, but not both.
```

Comment:
Will be good if can describe how the sort by policy command displays the list. Maybe add an example.

Reply:
Can you clarify what you mean by this?

Reply:
Like a screenshot of what happens when you perform "sort -i -dsc".